File: TYPESCRIPT PROGRAMMING/Test49.ts
// 49. PRINT AND CHECK WHETHER THE NUMBER IS PERFECT SQUARE OR NOT
const perfectSquare = (ip: number): void => {
    let sqrtIp: number = Math.floor(Math.sqrt(ip));
    let testIp: number = sqrtIp * sqrtIp;
    if(testIp === ip) {
        process.stdout.write(`${ip} Is A Perfect Square Number.`);
    } else {
        process.stdout.write(`${ip} Is Not A Perfect Square Number.`);  
    }
    console.log();
}
perfectSquare(16);
perfectSquare(8);
perfectSquare(65);
perfectSquare(100);

File: JAVA PROGRAMMING/Test49.java
// 49. PRINT AND CHECK WHETHER THE NUMBER IS PERFECT SQUARE OR NOT
import java.util.Scanner;
public class Test {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter IP Value: ");
        int ip = in.nextInt();
        int sqrt_ip = (int)(Math.sqrt(ip));
        System.out.println(sqrt_ip);
        int testip = sqrt_ip * sqrt_ip;
        System.out.println(testip);
        if(testip == ip) {
            System.out.println(true);
        } else {
            System.out.println(false);
        }
        in.close();

    }
}